client/components/banner/index.jsx

```diff
@@ -246,6 +248,7 @@ export class Banner extends Component {
 							) ) }
 						</ul>
 					) }
+					{ extraContent && <>{ extraContent }</> }
```

Comment:
Non-blocking:

Is this not the same as `{ extraContent }`? If the value is empty, nothing will be rendered; if non-empty, the value will be rendered as normal.

Reply:
Agree. I have made the changes: https://github.com/Automattic/wp-calypso/pull/81173/commits/a14a2af29983355f19a3d8da28314a35a46fde11